app/views/layouts/doorkeeper/application.html.erb

```diff
@@ -5,7 +5,8 @@
     <%= react_component('layout/Header', {
                           report_mode: ADMIN_OPTIONS['report_mode'],
                           version: ADMIN_OPTIONS['version'],
-                          show_demo_warning: ENV['SHOW_DEMO_WARNING'].present?,
+                          show_demo_warning_bg: ENV['SHOW_DEMO_WARNING_BG'].present?,
+                          demo_warning_msg: ENV['DEMO_WARNING_MSG'],
```

Comment:
Oh also, let's just call this "BANNER_MESSAGE" so it's more generic than "demo warning"

Reply:
the reason there's 2 values is because the demo env has a demo message and a demo background whereas the prod env has a demo message but no background

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1219/commits/309b77007688d1eec7240232fe3beaea230689c4